src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java

```diff
@@ -92,7 +91,6 @@ public AnnotationContext doAnnotate(AnnotationContext collectedInfo) {
   }
 
   private void addAnnotation(LocalIssuePointer i, AnnotationHolder annotationHolder) {
```

Comment:
This was not in the pull request. But I think `i` is best limited to counting loops, and should not be used anywhere else. I'd go as far as making that a rule.